src/components/icons/node-icon.js

```diff
@@ -17,9 +17,13 @@ export const paths = {
 };
 
 export default ({ className, type }) => (
-  <svg className={className} viewBox="0 0 24 24">
-    {paths[type].map(d => (
-      <path key={d} d={d} />
-    ))}
-  </svg>
+  <>
+    {paths[type] && (
+      <svg className={className} viewBox="0 0 24 24">
+        {paths[type].map(d => (
+          <path key={d} d={d} />
+        ))}
+      </svg>
+    )}
+  </>
```

Comment:
This might be better:
```es6
export default ({ className, type }) => paths[type] && (
  <svg className={className} viewBox="0 0 24 24">
    {paths[type].map(d => (
      <path key={d} d={d} />
    ))}
  </svg>
);
```

Reply:
Yeah unfortunately React complains if you don't return something, so I took the suggestion but tweaked it to return an empty node to satisfy it.

Reply:
ok in that case I'd typically use a ternary and return null if `!paths[type]`

Reply:
Sure I've done this, I guess React is fine with `null` but not `undefined`.